src/pages/ChokKahYang/landingKY.jsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import BaseKY from "./baseKY";
+
+export default class LandingKY extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            name: "KY Chok"
+
+        };
+
+    }
+
+    render() {
+        return (
+            <BaseKY title={"Chok Kah Yang (KY)"}>
```

Comment:
good try but please reuse component (Basepage in this case) if possible. Less code = less maintain, less file size